### PR TITLE
Added compatibility with metadata-service adaptor feature (closes #17)

### LINKED ISSUE #17: Add a way to redirect specific URLs to embeddable third-party endpoints (AKA: 'add-ons')
I suggest we start by having a hard-coded list of URL patterns/regex that can later become dynamic; putting the user/community in control of customising their experience.

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@arcturus r?

**Comment by wilsonpage** [comment]
This should land at about the same time as https://github.com/mozilla-magnet/magnet-metadata-service/pull/20

File: lib/scanner/ble/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class ScannerBle {
       .addListener('magnet:bledevicefound', this.onDeviceFound);
 
     // dummy beacons
-    // this.callback('https://vimeo.com/152985022');
+    this.callback('https://facebook.com/mozilla');
```
Comment (arcturus) [inline lib/scanner/ble/index.js]: nit: we dont need this :)

**Comment by arcturus** [comment]
r+ just clear the fake beacon and land please!

File: config.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "metadataServiceUrl": "http://tengam.org/api/v1/metadata",
```
Comment (arcturus) [inline config.json]: We should use the https version

**Pull request merged**